### DISCUSSION TITLE
Activity Report

### DISCUSSION THREAD

**Discussion by AkshatGoyal1018** [Newbyes/Hackerrank#2] [General]
@everyone who is taking part in this program please try to update here whenever you complete a code and commits.
So this way everyone can get a track of everyone and can get motivated by each other. Thank You !

**Comment by PHAGUN-JAIN** [comment]
> 
> 
> @everyone who is taking part in this program please try to update here whenever you complete a code and commits.
> So this way everyone can get a track of everyone and can get motivated by each other. Thank You !

@AkshatGoyal1018 What do you mean "aisa nahi kiya toh bahar fak diye jaaoge yeh koi baat hui bhala" XD

**Reply by AkshatGoyal1018** [reply]
When did I told you about my intentions. xD 
Although I won't do that.

**Comment by AkshatGoyal1018** [comment]
I pushed Two codes today of hackerrank 
Day and Time 
Currency
